Makefile

```diff
@@ -268,9 +268,9 @@ localnet-start: localnet-stop
 	docker rmi -f ethermint-build-linux
 	docker build --rm -t ethermint-build-linux . ; \
 	container_id=$$(docker create ethermint-build-linux) ; \
-	docker cp $${container_id}:/usr/bin/emintd ./build/ ; \
-	docker cp $${container_id}:/usr/bin/emintcli ./build/
-	if ! [ -f build/node0/$(ETHERMINT_DAEMON_BINARY)/config/genesis.json ]; then docker run --rm -v $(CURDIR)/build:/$(ETHERMINT_DAEMON_BINARY):Z emintd/node testnet --v 4 -o . --starting-ip-address 192.168.10.2 --keyring-backend=test ; fi
+	docker container start $${container_id} ;
+
+	if ! [ -f build/node0/$(ETHERMINT_DAEMON_BINARY)/config/genesis.json ]; then docker run --rm -v $(CURDIR)/build:/ethermint:Z ethermint-build-linux "emintd testnet --v 4 -o ethermint --starting-ip-address 192.168.10.2 --keyring-backend=test"; fi
```

Comment:
> `$(CURDIR)/build:/ethermint:Z`

I don't think the `ethermint` directory exists

Reply:
the directory is mounted to the volume on your local machine ($CURDIR/build) and the `ethermint` dir is created when mounted